.github/CONTRIBUTING.md

```diff
@@ -17,6 +17,25 @@ To get ready to work on the codebase, please do the following:
 5. Run `yarn test` to run ESLint and ensure any JSDoc changes are valid
 6. [Submit a pull request](https://github.com/discordjs/discord.js/compare) (Make sure you follow the [conventional commit format](https://github.com/discordjs/discord.js/blob/main/.github/COMMIT_CONVENTION.md))
 
+## Testing changes locally
+
+If you want to test changes you've made locally, you can do so by using `yarn link`. This will create a symlink to your local copy of the discordjs libraries.`
```

Comment:
Trailing backtick as well as opting for "discord.js" terminology:

```suggestion
If you want to test changes you've made locally, you can do so by using `yarn link`. This will create a symlink to your local copy of the discord.js libraries.
```